tests/rados/test_track_availibility_status.py

```diff
@@ -0,0 +1,604 @@
+        log.info(
+            "=== Case 11: Verification enable_availability_tracking parameter completed  ==="
+        )
+        log.info(
+            "=== Case 12: Verification clear-availability-status command on pool  ==="
+        )
+
+        cmd_clear_pool_status = f"ceph osd pool clear-availability-status {pool_name}"
+        client.exec_command(cmd=cmd_clear_pool_status)
+        out_put = get_pool_availability_status(client)
+        selected_pool_details = out_put[pool_name]
+        if (
+            int(selected_pool_details[2]) != 0
+            and int(selected_pool_details[4]) != 1
+            and int(selected_pool_details[5]) != 1
+        ):
+            msg_error = f"The clear_availability_status command  not clear the {pool_name} pool data"
+            log.error(msg_error)
+            return 1
+        msg_info = (
+            f"The clear_availability_status command clear the {pool_name} pool data"
+        )
+        log.info(msg_info)
+        log.info(
+            "=== Case 12: Verification clear-availability-status command on pool completed  ==="
+        )
+
+    except Exception as e:
+        log.info(e)
+        log.info(traceback.format_exc())
+        return 1
+    finally:
+        log.info(
+            "\n \n ************** Execution of finally block begins here *************** \n \n"
+        )
+        # if config.get("delete_pool"):
+        #     method_should_succeed(rados_object.delete_pool, pool_name)
+        mon_obj.set_config(
+            section="mon", name="enable_availability_tracking", value="true"
+        )
+    return 0
+
+
+# ceph osd pool availability-status
+
+
+def get_pool_availability_status(client):
+    """
+    Method is used to get the availability status of cluster
+    Args:
+        client : client object
+    Returns:
+        available_data_in_dict :Available data in dictionary.
+        For example:
+              POOL             UPTIME  DOWNTIME    NUMFAILURES  MTBF  MTTR  SCORE  AVAILABLE
+              .mgr                 4d     0s            0        0s    0s      1          1
+          Output : [.mgr] ->['4d','0s','0','0s','0s','1','1']
+    """
+    cmd = "ceph osd pool availability-status"
+    cmd_out_put = ""
+    try:
+        cmd_out_put = client.exec_command(cmd=cmd)
+    except Exception as err:
+        log.error(err)
+        err_msg = (
+            "Error ENOTSUP: availability tracking is disabled; you can enable it by setting the config option "
+            "enable_availability_tracking"
+        )
+        if err_msg in str(err):
+            return err_msg
+
+    exact_data = cmd_out_put[0]
+
+    # Split into lines and skip the header
+    lines = exact_data.strip().split("\n")[1:]
+    available_data_in_dict = {}
+    for line in lines:
+        parts = line.split()
+        pool_name = parts[0]
+        available_data_in_dict[pool_name] = tuple(parts[1:])
+    return available_data_in_dict
+
+
+def time_to_seconds(time_str: str) -> int:
```

Comment:
make use of builtins to achieve this

Reply:
Used timedelta to covert the time into seconds